lib/jekyll-compose/arg_parser.rb

```diff
@@ -24,4 +25,8 @@ def title
   def force?
     !!options["force"]
   end
+
+  def source
+    source = config['source'].gsub(/^#{Regexp.quote(Dir.pwd)}/, '')
```

Comment:
I've updated this and removed all unnecessary variable assignments in that class.